client/tailscale/localclient.go

```diff
@@ -825,6 +829,25 @@ func (lc *LocalClient) GetDNSOSConfig(ctx context.Context) (*apitype.DNSOSConfig
 	return &osCfg, nil
 }
 
+// PerformDNSQuery exexutes a DNS query for the given name and query type. It returns the
+// raw DNS response bytes and the resolvers that were used to answer the query (often just
+// one, but can be more if we raced multiple resolvers).
+func (lc *LocalClient) PerformDNSQuery(ctx context.Context, name string, queryType dnsmessage.Type) (bytes []byte, resolvers []*dnstype.Resolver, err error) {
```

Comment:
I think I'd prefer this take the string form of the dnsmessage.Type instead ("A", "NS", "AAAA").

And you can say that for other types, it can be a decimal number ("123").